Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Utilities/RotateToValue.cs

```diff
@@ -0,0 +1,228 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace HoloToolkit.Examples.Prototyping
+{
+    /// <summary>
+    /// animates the rotation of an object with eases
+    /// </summary>
+    public class RotateToValue : MonoBehaviour
+    {
+        /// <summary>
+        /// ease types
+        ///     Linear: steady progress
+        ///     EaseIn: ramp up in speed
+        ///     EaseOut: ramp down in speed
+        ///     EaseInOut: ramp up then down in speed
+        ///     Free: super ease - just updates as the TargetValue changes
+        /// </summary>
+        public enum LerpTypes { Linear, EaseIn, EaseOut, EaseInOut, Free }
```

Comment:
Rename:
LerpTypes->EaseTypes
EaseIn -> In
EaseOut -> Out
EaseInOut -> InOut

Reply:
By guidelines, should this be it's own file? Or at least outside of this class since it is public.